### ISSUE TITLE
Problem with tags (not?) being case-sensitive

### ISSUE THREAD

**Issue by ocrasborn** [Woseseltops/signbank#111]
It would be nice if tags were case-sensitive. Now it appears they are not: new tags are converted to all-lowercase. There is however one existing tag with an uppercase letter, Radboud_sign. But when I apply that to a new sign, the system creates a new tag that is all-lowercase. So on the one hand the tags appear case-sensitive (one uppercase letter is saved and correctly displayed), but applying it to new signs leads to the creation of a lowercase variant.

**Comment by Woseseltops** [comment]
This was incredibly easy to fix, but incredibly hard to find. The tagging functionality is provided by an external module, so I couldn't directly edit the code. However, Google told me that the behavior to force lowercase was not standard, so there must be some code somewhere that made everything lowercase (or so I thought). I looked at numerous places, doing tests to pinpoint the exact location where it happened, and was about to give up... when I suddenly noticed a setting in the settings file 'FORCE_LOWERCASE_TAGS'. When I set that to False, everything worked :P.

I also removed the lowercase variant of Radboud_sign and added the uppercase one here http://signbank.science.ru.nl/dictionary/gloss/3269/
